packages/adp-tooling/src/types.ts

```diff
@@ -56,6 +69,32 @@ export interface AdpWriterConfig {
     };
 }
 
+export interface ChangeInboundNavigation {
+    /** Identifier for the inbound navigation. */
+    inboundId: string;
+    title?: string;
+    subTitle?: string;
+}
+
+export interface NewInboundNavigation {
+    /** Represent business entities and can bundle applications that reflect a specific scenario. */
+    semanticObject: string;
+    /** Operations which can be performed on a semantic object. */
+    action: string;
+    //** Defined instance of the semantic object (e.g. by specifying the employee ID). */
+    additionalParameters?: object;
+    title: string;
+    subTitle?: string;
+    inboundId?: string;
```

Comment:
`inboundId` shouldn't exit on a new navigation, correct?

Reply:
Moved in internal type